### ISSUE TITLE
additional parameters for mailman-extra.cfg / site_owner

### ISSUE THREAD

**Issue by fsantiago07044** [maxking/docker-mailman#166]
in mailman-extra.cfg, you should also tell admins to add:

```
[mailman]
# This address is the "site owner" address.  Certain messages which must be
# delivered to a human, but which can't be delivered to a list owner (e.g. a
# bounce from a list owner), will be sent to this address.  It should point to
# a human.
site_owner: changeme@example.com
```

and change it to their own valid address.

**Comment by maxking** [comment]
This should ideally be a part of the docs. I will do that.

**Comment by brianemwd** [comment]
Did this ever make it to the documentation?

**Comment by maxking** [comment]
Not yet, do you want to submit a MR for it?

**Comment by brianemwd** [comment]
I would if I knew what a MR is. I am very new (or late) to the Github service. Mailman 3 is the first application that I have had interest in.

 

Have a great day!

 

Brian Carpenter

EMWD.com

brian@emwd.com

**Comment by maxking** [comment]
I guess MR is a wrong term for Github too.

Pull Request is Github's way to sending a patch to maintainers.

https://help.github.com/en/articles/editing-files-in-your-repository

This is a simple way to edit file and propose it to be merged here right in the browser.  Hit the :memo: icon on the top right of https://github.com/maxking/docker-mailman/blob/master/README.md page and then you will be able to edit the file and submit a Pull Request or PR.

**Comment by brianemwd** [comment]
Ok, I will give that a try this week. I can also try to help out and test Xapian as a search engine. I will install your latest images on a server and set it up and document the installation process. I might need some technical guidance from you regarding that. 

 

I figured out what a MR is: merge request.

 

Have a great day!

 

Brian Carpenter

EMWD.com

brian@emwd.com

**Comment by brianemwd** [comment]
Hi Abhilash,

I believe I submitted the PR correctly.

**Comment by maxking** [comment]
Fixed by #328
